app/BUILD.bazel

```diff
@@ -336,6 +388,70 @@ kt_android_library(
     visibility = ["//visibility:private"],
 )
 
+# Library for all custom data-binding adapters under org/oppia/app/databinding
+android_library(
+    name = "databinding_adapters",
+    custom_package = "org.oppia.app.impl.adapters",
+    srcs = DATABINDING_ADAPTER_FILES,
+    enable_data_binding = 1,
+    manifest = "src/main/DbAdaptersAndroidManifest.xml",
+    deps = [
+        ":dagger",
+        "//model",
+        "//utility",
+        ":view_models",
+        ":resources",
```

Comment:
Please keep the ordering consistent, specifically I believe local libraries are listed before top-level ones, and those should be before external workspace imports which are before artifacts.

Ditto elsewhere.

Reply:
Fixed!